tests/ast/decoder.py

```diff
@@ -766,6 +766,20 @@ def decode_expr(self, expr: proto.Expr) -> Any:
                         block=block,
                         case_sensitive=case_sensitive,
                     )
+            case "sp_dataframe_count":
+                df = self.symbol_table[expr.sp_dataframe_count.id.bitfield1][1]
+                d = MessageToDict(expr.sp_dataframe_count)
+                statement_params = {}
+                statement_params_list = d.get("statementParams", [])
+                for statement_params_list_map in statement_params_list:
```

Comment:
might be a good idea to make a helper function to process this logic since this was also used with `collect`